test/test_ops.py

```diff
@@ -59,6 +59,27 @@ def test_supported_dtypes(self, device, dtype, op):
             self.assertTrue(sample_input.dtype == dtype)
             self.assertTrue(sample_input.device.type == self.device_type)
 
+    # Verifies that backward for each unsupported floating or complex dtype
+    #   throw a runtime error.
+    @onlyOnCPUAndCUDA
+    @ops(op_db, dtypes=OpDTypes.unsupported_backward,
+         allowed_dtypes=floating_and_complex_types_and(torch.float16, torch.bfloat16))
+    def test_unsupported_backward(self, device, dtype, op):
```

Comment:
making 40+ test skips is not fun and I've been chasing moving targets. 
For some reason not ALL test cases fails and raises RuntimeError for some unsupported backward operators. 

The initial intend of this test is to "make sure when backward support was added for an operator/dtype, they should be covered by the test case" - thus properly adding the operator/dtype to the test list and remove from the skip list

For this intent, I think we can simply expect to catch ANY RuntimeError when ran against samples instead of requiring ALL of them to raise RuntimeError. Please let me know what do you think of this idea @mruberry

Reply:
I agree with you and elaborate on the philosophy for these tests below.

Because this is likely to hit both logical and physical merge conflicts I suggest merging it outside business hours (unfortunately).